index.html

```diff
@@ -0,0 +1,368 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Form Factors Explained</title>
+  <script src="https://www.w3.org/Tools/respec/respec-w3c" class="remove" defer></script>
+  <script class="remove">
+   // All config options at https://respec.org/docs/ 
+   var respecConfig = {
+      subtitle: "Foldable devices and multi-screen setups",
+      specStatus: "unofficial",
+      editors: [
+        { name: "Second Screen Community Group", url: "https://www.w3.org/community/webscreens/" },
+      ],
+      github: "webscreens/form-factors",
+      shortName: "form-factors",
+      xref: "web-platform",
+      group: "webscreens",
+      format: "markdown"
+    };
+  </script>
+</head>
+<body>
+
+<section id="abstract">
+## Abstract
+
+This explainer discusses new device [=form factors=] that are emerging and related terminology that is being established. This document proposes one naming scheme to be able to reason about and use these terms and concepts consistently in [related web specifications](#related-web-specifications). The use of these terms in other contexts may differ.
```

Comment:
not wrapping these lines makes it really hard to review on github

Reply:
I propose use [Preview](https://raw.githack.com/webscreens/form-factors/first-stab/index.html).

Tidy does not work with markdown, sadly. I'm open to PRs if someone wants to manually wrap the lines :)